kaspresso/src/main/kotlin/com/kaspersky/kaspresso/runner/listener/KaspressoLateRunListener.kt

```diff
@@ -0,0 +1,31 @@
+package com.kaspersky.kaspresso.runner.listener
+
+import org.junit.runner.Description
+
+interface KaspressoLateRunListener : KaspressoRunListener {
```

Comment:
`KaspressoRunListener`s are being set to runner when creating `Kaspresso.Builder`. At this moment `testRunStarted` and `testStarted ` are already called, but the listener may need to handle these events little bit later.